models/yang/annotations/sonic-showtech-annot.yang

```diff
@@ -0,0 +1,17 @@
+module sonic-showtech-annot {
```

Comment:
why these yang model not in sonic-yang-model in build image?

Reply:
We use RPC (to use internal Click command via d-bus infra) for show-tech operation from mgmt-framework and no config-DB update here and hence no YANG model is present in sonic-yang-model in build image.